samples/nrf9160/modem_trace_backend/Kconfig.trace_print_stats

```diff
@@ -11,9 +11,9 @@ choice NRF_MODEM_LIB_TRACE_BACKEND
 
 config NRF_MODEM_LIB_TRACE_BACKEND_CUSTOM
 	bool "Count modem traces and print to terminal"
-	select CPU_LOAD
-	select FPU
-	select NEWLIB_LIBC_FLOAT_PRINTF
+	depends on CPU_LOAD
```

Comment:
Can this change (`select` to `depends on`) be moved to a separate commit since it's separate from the introduction of the flash backend?

Reply:
done